ios/Classes/BetterPlayerPlugin.m

```diff
@@ -155,13 +157,18 @@ - (void) setRemoteCommandsNotificationActive{
 }
 
 - (void) setRemoteCommandsNotificationNotActive{
-    if ([_players count] == 0) {
-        [[AVAudioSession sharedInstance] setActive:false error:nil];
-    }
-
+    [[AVAudioSession sharedInstance] setActive:false error:nil];
     [[UIApplication sharedApplication] endReceivingRemoteControlEvents];
 }
 
+- (void) removeCommandCenterTargetHandlers{
+    MPRemoteCommandCenter *commandCenter = [MPRemoteCommandCenter sharedCommandCenter];
+    [commandCenter.togglePlayPauseCommand removeTarget:_commandCenterTargetIdDict[@"togglePlayPauseCommand"]];
```

Comment:
Sorry, I don't have enough experience for using `MPRemoteCommand`.

I checked the document of `removeTarget`.
https://developer.apple.com/documentation/mediaplayer/mpremotecommand/1622903-removetarget
It says,
> Specify nil to remove all targets.

We cannot specify `nil`? Please check it 🙏

Reply:
@noboru-i 
Oh sorry, my mistake, i'm using `nil` now, and it work the same, could you please review it again 🙇?